spec/integration/federation/federation_helper.rb

```diff
@@ -21,10 +21,10 @@ def allow_public_key_fetch(user)
 end
 
 def create_undiscovered_user(pod)
-  FactoryGirl.build(:user).tap do |user|
+  FactoryBot.build(:user).tap do |user|
     allow(user).to receive(:person).and_return(
-      FactoryGirl.build(:person,
-                        profile:               FactoryGirl.build(:profile),
+      FactoryBot.build(:person,
+                        profile:               FactoryBot.build(:profile),
```

Comment:
Layout/ArgumentAlignment: Align the arguments of a method call if they span more than one line.